templates/search/includes/filter.html

```diff
@@ -16,4 +16,8 @@
             <input type="submit" value="Update" class="search-filters__submit">
         </fieldset>
     </div>
+{% elif field.data and not field.errors %}
+    {% for value in field.data %}
+        <input type="hidden" name="{{ field.name }}" value="{{ value }}">
+    {% endfor %}
```

Comment:
Question: Why is this here and what scenario it covers?

Reply:
Previously, you had some [strange code within the date filter output](https://github.com/nationalarchives/ds-wagtail/blob/develop/templates/search/blocks/search_filters.html#L94-L95) to account for this, but this is a better solution to that problem.

Reply:
Ok, That bit of code was not introduced in the [PR](https://github.com/nationalarchives/ds-wagtail/pull/991/files) but I've explained and removed/adjusted them with  `render_form_fields_as_hidden` for the filter section of the form.